controller/controller.go

```diff
@@ -206,8 +206,12 @@ func Install(rootDir string, model *model.SystemInstall, options args.Args) erro
 
 	// prepare all the target block devices
 	for _, curr := range model.TargetMedias {
+		var wholeDisk bool
+		if val, ok := model.InstallSelected[curr.Name]; ok {
```

Comment:
Is this necessary? Could we pass `model.InstallSelected[curr.Name]` as a parameter?